packages/components/pie-modal/src/index.ts

```diff
@@ -102,21 +105,21 @@ export class PieModal extends RtlMixin(LitElement) implements ModalProps {
         this.addEventListener('click', (event) => this._handleDialogLightDismiss(event));
     }
 
-    connectedCallback () : void {
+    connectedCallback (): void {
```

Comment:
certainly intrigued as to why the linter either:

- Doesn't flag these
- Doesn't fix these
- Accepts both the previous and new formats as valid (i.e your editor config differs from someone else's but it thinks both are fine).

It feels like we're playing whitespace ping pong between PRs at the moment 😂